pkg/scope/machine.go

```diff
@@ -104,6 +105,36 @@ func (m *MachineScope) PatchObject(ctx context.Context) error {
 	return m.patchHelper.Patch(ctx, m.HivelocityMachine)
 }
 
+// DeviceTagMachine returns a DeviceTag object for the machine tag.
+func (m *MachineScope) DeviceTagMachine() hvtag.DeviceTag {
```

Comment:
Somehow I think `NewDeviceTag()` might be a bit easier to understand. "New" signals "this is creating a new struct". And the struct is "DeviceTag".

But the name `DeviceTagMachine()` is ok, too.

Reply:
Moved the method - it is now defined as HivelocityMachine.DeviceTag()